modules/visualization/dev-instance-foxbox/LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
There is a LICENSE available in the root of ADDF repo. You can delete this here.